packages/@vuepress/theme-default/components/DropdownLink.vue

```diff
@@ -68,9 +73,38 @@ export default {
     }
   },
 
+  computed: {
+    allItemsCount () {
+      return this.item.items.reduce((totalCount, item) => {
+        return item.items ? totalCount + item.items.length : totalCount + 1
+      }, 0)
+    },
+
+    dropdownAriaLabel () {
+      return this.item.ariaLabel || this.item.text
+    }
+  },
+
   methods: {
     toggle () {
       this.open = !this.open
+    },
+
+    getSubItemIndex (index, subIndex) {
```

Comment:
This part is kind of hard to read/understand, I'll try to find something easier for this part

Reply:
I too will try to find to a more easy solution.